packages/wallets/webpack.config.js

```diff
@@ -0,0 +1,189 @@
+                                keepQuery: true,
+                            },
+                        },
+                        'sass-loader',
+                        {
+                            loader: 'sass-resources-loader',
+                            options: {
+                                // Provide path to the file with resources
+                                resources: [
+                                    // eslint-disable-next-line global-require, import/no-dynamic-require
+                                    ...require('../shared/src/styles/index.js'),
+                                    // eslint-disable-next-line global-require, import/no-dynamic-require
+                                    ...require('./src/styles/index.js'),
+                                ],
+                            },
+                        },
+                    ],
+                },
+                {
+                    test: /\.svg$/,
+                    issuer: /\/packages\/wallets\/.*(\/)?.*.scss/,
+                    exclude: /node_modules/,
+                    include: /public\//,
+                    type: 'asset/resource',
+                    generator: {
+                        filename: 'wallets/public/[name].[contenthash][ext]',
+                    },
+                },
+                {
+                    test: /\.svg$/,
+                    issuer: /\/packages\/wallets\/.*(\/)?.*.tsx/,
+                    exclude: /node_modules/,
+                    include: /public\//,
+                    use: svg_loaders,
+                },
+            ],
+        },
+        optimization: {
+            minimize: is_release,
+            minimizer: is_release
+                ? [
+                      new TerserPlugin({
+                          test: /\.js$/,
+                          parallel: 2,
+                      }),
+                      new CssMinimizerPlugin(),
+                  ]
+                : [],
+            splitChunks: {
+                chunks: 'all',
+                minSize: 102400,
+                minSizeReduction: 102400,
+                minChunks: 1,
+                maxAsyncRequests: 5,
+                maxInitialRequests: 3,
+                automaticNameDelimiter: '~',
+                enforceSizeThreshold: 500000,
+                cacheGroups: {
+                    default: {
+                        minChunks: 2,
+                        minSize: 102400,
+                        priority: -20,
+                        reuseExistingChunk: true,
+                    },
+                    defaultVendors: {
+                        idHint: 'vendors',
+                        test: /[\\/]node_modules[\\/]/,
+                        priority: -10,
+                    },
+                    shared: {
+                        test: /[\\/]shared[\\/]/,
+                        name: 'shared',
+                        chunks: 'all',
+                    },
```

Comment:
do we use shared in wallets?

Reply:
No, but since we are using `@deriv/api` and it uses shared 👀